### PR TITLE
Add survey_id to v2 launch pattern

### PR DESCRIPTION
**Pull request by Hajara-I**
### What is the context of this PR?
Add survey_id to v2 launch pattern to support CIR

### CODE REVIEW

**Comment by Hajara-I** [comment]
[Add survey_id to v2 launch pattern to support CIR (1 SP)](https://trello.com/c/s6KQ9que/5582-add-surveyid-to-v2-launch-pattern-to-support-cir-1-sp)

**Review by Yuyuutsu — COMMENTED** [review]
Changes look good. Do you need to have it the same as validator?

File: docs/rm_to_eq_runner_payload_v2.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ The data property must adhere to one of [Business Survey Metadata][business_surv
 ##### Business Survey Metadata
 
 | **Property**         | **Definition**                                                                                                                                                                |
-| -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|----------------------| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (rmccar) [inline docs/rm_to_eq_runner_payload_v2.md:77]: This change doesn't fit with the rest of the table

```suggestion
| -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

File: schemas/common/definitions.json [hunk 2]
```diff
@@ -198,7 +198,9 @@
   },
   "survey_id": {
     "type": "string",
-    "description": "The survey identifier as used across the ONS.",
+    "description": "The ONS id of the survey derived from the inquiry code already in use for that survey.",
+    "pattern": "^[0-9a-z]+$",
+    "minLength": 1,
```
Comment (rmccar) [inline schemas/common/definitions.json:203]: Is the minimum length 1? I would've thought it would be more than that?

Comment (Hajara-I) [reply schemas/common/definitions.json:203]: Yes, Mebin suggested making it `minLength: 1` just to make sure it's not an empty string.

Comment (rmccar) [reply schemas/common/definitions.json:203]: Fair enough. Looks good then other than my other comment

**Review by rmccar — APPROVED** [review]

**Review by pricem14pc — APPROVED** [review]
This is as expected following the most recent discussions wit @MebinAbraham

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**